app/state_machines/efile_submission_state_machine.rb

```diff
@@ -120,21 +120,26 @@ class EfileSubmissionStateMachine
     tax_return.transition_to(:file_accepted)
 
     benefits = Efile::BenefitsEligibility.new(tax_return: tax_return, dependents: submission.qualifying_dependents)
-    ctc_amount = benefits.ctc_amount / 2
-    refund_amount = client.intake.claim_owed_stimulus_money_no? ? 0 : (benefits.eip1_amount + benefits.eip2_amount)
-    eip3_amount = benefits.eip3_amount
+    eip1_and_eip2_amount = [benefits.eip1_amount, benefits.eip2_amount].compact.sum
+    total_refund_amount = [benefits.outstanding_ctc_amount, benefits.outstanding_recovery_rebate_credit].compact.sum
+    eip3_amount_received = tax_return.intake.eip3_amount_received || 0
```

Comment:
Alternatively, we can set a database default for this value to 0 OR create a custom reader on the model that reads from the db value if present or returns 0.

Reply:
i think putting it on benefits may make the most sense, since in the context of the intake there may be some logic that distinguishes between a 0 and nil value somewhere